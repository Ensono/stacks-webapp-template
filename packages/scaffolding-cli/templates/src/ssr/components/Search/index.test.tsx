import {render} from "@testing-library/react"
import React from "react"
import {Search} from "."

test("renders ApiPane snapshot", () => {
    const {asFragment} = render(<Search getSearchResults={() => {}} />)
    expect(asFragment()).toMatchSnapshot()
})
