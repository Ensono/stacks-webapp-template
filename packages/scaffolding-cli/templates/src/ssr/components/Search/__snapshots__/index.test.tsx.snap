// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ApiPane snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiContainer-root MuiContainer-maxWidthSm"
  >
    <div
      class="MuiBox-root MuiBox-root-10"
    >
      <div
        class="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal MuiFormControl-fullWidth"
        data-testid="search_btn"
      >
        <label
          class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
          data-shrink="false"
          for="search-bar"
          id="search-bar-label"
        >
          Search Menus
        </label>
        <div
          class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
        >
          <input
            aria-describedby="search-bar-helper-text"
            aria-invalid="false"
            class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputTypeSearch"
            id="search-bar"
            name="SearchTerm"
            type="search"
            value=""
          />
          <fieldset
            aria-hidden="true"
            class="PrivateNotchedOutline-root-69 MuiOutlinedInput-notchedOutline"
          >
            <legend
              class="PrivateNotchedOutline-legendLabelled-71"
            >
              <span>
                Search Menus
              </span>
            </legend>
          </fieldset>
        </div>
        <p
          class="MuiFormHelperText-root MuiFormHelperText-contained"
          id="search-bar-helper-text"
        >
          Search terms are case sensitive and partial matches are not searched
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
